package.json

```diff
@@ -9,7 +9,8 @@
   },
   "scripts": {
     "test": "test",
-    "cli": "node lib/index.js"
+    "cli": "node lib/index.js",
+    "build": "npx tsc"
```

Comment:
tsc samo wystarczy, `npx` się używa bardziej w CLI i lokalnie, tak jak coś jest npm scriptem to tak jakby automatycznie dostaje npx